graphql/src/graphql_schema.ml

```diff
@@ -437,14 +433,6 @@ module Make (Io : IO) (Stream: Stream with type 'a io = 'a Io.t) = struct
     | _ ->
         invalid_arg "Arguments must be Interface/Union and Object"
 
-  let obj_of_subscription_obj {name; doc; fields} =
-    let fields = List.map
-      (fun (SubscriptionField {name; doc; deprecated; typ; args; resolve}) ->
-        Field { lift = Obj.magic (); name; doc; deprecated; typ; args; resolve = (fun ctx () -> resolve ctx) })
-      fields
-    in
-    { name; doc; abstracts = ref []; fields = lazy fields }
-
```

Comment:
What are your thoughts on moving these lines (and the ones above) further down?

Reply:
Where exactly? I've moved them just before the introspection module which feels ok to me.

Reply:
Sorry, I was being unclear: why move them at all in this PR? 😄

Reply:
Oh we actually don't need to. I was envisioning passing some directives argument to `schema`, which would mean their definition would have to come before.